### PR TITLE
Various text changes

### PR DESCRIPTION
**Pull request by lostcause120**
Various text changing ranging from minor to major changes to text lines

Minor Change Examples: 
- adding parenthesis for inner rain thoughts
- pronoun consistency changes
- changing newline locations to prevent text from being cut off
etc...
Major Change Examples:
- changing sentence structure for better clarity
- changing menu text for consistency
- getting rid of -chan, -san - and -sama everywhere( I replaced sama with some titles, but -chan and -san were just deleted)
- adding translations for random small things that were missed
etc...

### CODE REVIEW

**Comment by lostcause120** [comment]
textfile with original text along with change below them
[textchanges.txt](https://github.com/DazedMTL/NightmareKnight/files/13767916/textchanges.txt)

File: .gitignore
Comment (dazedanon) [inline .gitignore:1]: These images need to be left as is.

**Review by dazedanon — CHANGES_REQUESTED** [review]
Everything looks good, just have the gitignore include the pictures and I'll merge it.

**Review by dazedanon — APPROVED** [review]
Actually I'll just merge and make the gitignore changes myself.

File: .gitignore
Comment (lostcause120) [reply .gitignore:1]: Ah thanks, wasn't sure which images you referring to and still not familiar with github.

**Pull request merged**